pkgs/development/interpreters/renpy/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, stdenv, fetchFromGitHub, python3Packages, pkg-config, SDL2
+, libpng, ffmpeg, freetype, glew, libGL, libGLU, fribidi, zlib
+, glib
+}:
+
+with python3Packages;
+
+stdenv.mkDerivation rec {
+  pname = "renpy";
+  version = "8.0.0";
+
+  meta = with lib; {
+    description = "Ren'Py Visual Novel Engine";
+    homepage = "https://renpy.org/";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ shadowrz ];
+  };
+
+  src = fetchFromGitHub {
+    owner = "renpy";
+    repo = "renpy";
+    rev = "8.0.0.22062402";
+    sha256 = "sha256-37Hbs0i5eXMjVaETX7ImJCak0y8XtEHUaRFceA9J39A=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    python cython wrapPython tkinter future six pefile requests
+    SDL2 libpng ffmpeg freetype glew libGLU libGL fribidi zlib pygame_sdl2 glib
```

Comment:
Are those actually used at compile time or only through wrapping in RENPY_DEPS_INSTALL? If the second we can drop them from here.

Reply:
They are used actually. I tried to drop them here, but the build system couldn't find the headers.